doc/pipeline.md

```
@@ -0,0 +1,29 @@
+# Transform Pipeline
+
+This document describes the image transformation pipeline that `transform_raster()` applies to raster inputs.
+
+## Pipeline stages
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)